apps/antalmanac/tests/calendarize-helpers.test.ts

```diff
@@ -0,0 +1,227 @@
+import { describe, test, expect } from 'vitest';
+import type { ScheduleCourse } from '@packages/antalmanac-types';
+import type { CourseEvent, CustomEvent } from '$components/Calendar/CourseCalendarEvent';
+import type { RepeatingCustomEvent } from '$components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog';
+import { calendarizeCourseEvents, calendarizeCustomEvents, calendarizeFinals } from '$stores/calendarizeHelpers';
+
+describe('calendarize-helpers', () => {
+    const courses: ScheduleCourse[] = [
+        {
+            courseComment: 'string',
```

Comment:
Can you make these more obviously placeholders, like "foobar" or "placeholder1" or something? I was confused for a couple of minutes looking at this because I thought it was a janky type definition 💀

Reply:
Lol, I took copied the type definition and used it as the example; so yeah it basically is a janky type def. I'll replace the values with more random placeholders.